test/unittests/test_builtins.cpp

```diff
@@ -141,3 +141,13 @@ TEST(builtins, is_constant_evaluated)
     EXPECT_EQ(nonconstexpr_func_res, 1);
 #endif
 }
+
+TEST(builtins, bswap)
+{
+    EXPECT_EQ(bswap(uint8_t{0x01}), 0x01);
+    EXPECT_EQ(bswap(uint16_t{0x6601}), 0x0166);
```

Comment:
Problem with writing runtime tests like this is that a compiler is allowed to compute the function result in compile-time even when building without optimizations.
The workaround is to place the argument in non-constexpr variable.
```suggestion
    uint16_t x = 0x6601;
    EXPECT_EQ(bswap(x), 0x0166);
```

Unfortunately, we don't have coverage from MSVC and old compilers to confirm non-constexpr path in the function is covered.

Here you can see how compilers handle such cases. Clang is evaluating at compile-time.
https://godbolt.org/z/1n8rW9xjs